### ISSUE TITLE
[1.17.1] #55 still not fixed

### ISSUE THREAD

**Issue by MSandro** [Dunkmania101/SpatialHarvesters#60]
#55
newest version for 1.17.1 is 0.11.12d
I can not connect any energy cable!

**Comment by zerebos** [comment]
Yes this is still an outstanding issue for multiple modpacks on 1.17.1. Hopefully It's as simple as @Dunkmania101 said https://github.com/Dunkmania101/SpatialHarvesters/issues/55#issuecomment-1022215002
